frontend/src/features/admin-form/create/logic/components/LogicContent/EditLogicBlock/EditCondition/EditConditionBlock.tsx

```diff
@@ -154,6 +154,10 @@ export const EditConditionBlock = ({
       case BasicField.YesNo:
         return ['Yes', 'No']
       case BasicField.Radio:
+        if (mappedField.othersRadioButton)
+          // 'Others' does not show up in fieldOptions
+          return mappedField.fieldOptions.concat('Others')
```

Comment:
nit (on behalf of KR lool): let's add braces to multiline `if` blocks 
```suggestion
        if (mappedField.othersRadioButton) {
          // 'Others' does not show up in fieldOptions
          return mappedField.fieldOptions.concat('Others')
        }
```

Reply:
HAHA ok